specification/frontdoor/resource-manager/readme.md

```diff
@@ -171,6 +171,10 @@ See configuration in [readme.go.md](./readme.go.md)
 These settings apply only when `--java` is specified on the command line.
 Please also specify `--azure-libraries-for-java-folder=<path to the root directory of your azure-libraries-for-java clone>`.
 
+## CLI
```

Comment:
Please don't put these CLI lines in the middle of Java configuration.

Reply:
sorry, my stupid mistake